### DISCUSSION TITLE
Discontinuation of _vacinas_detalhe.csv_ and its merge with _vacinas.csv_

### DISCUSSION THREAD

**Discussion by lynxgav** [dssg-pt/covid19pt-data#1148] [Dados Nacionais e por ARS]
Hi all,

thanks for the awesome repository that is really useful for researchers in the field of infectious disease modelling. We plan to use the data from _vacinas_detalhe.csv_ in the extension of our model [published](https://rdcu.be/cmDC4) in 2021 (*). 

Is there a reason that _vacinas_detalhe.csv_ was discontinued on 15 Nov 2021? Is it possible to continue updating it (with e.g. the addition of other columns for boosters)? It seems that now only _vacinas.csv_ is updated regularly but this file does not have the age stratification and pessoas variables as in _vacinas_detalhe.csv_. Therefore it is not possible to "merge" these two files/datasets in our analyses that require age stratification.

I look forward to hearing from you.

Bom 2022 and thanks again!

Kind regards,
Ganna Rozhnova

João Viana, Christiaan H. van Dorp, Ana Nunes, Manuel C. Gomes, Michiel van Boven, Mirjam E. Kretzschmar, Marc Veldhoen, and Ganna Rozhnova (2021). Controlling the pandemic during the SARS-CoV-2 vaccination rollout. Nature Communications 12, 3674 (2021). https://doi.org/10.1038/s41467-021-23938-8.

NB! In this publication, we also benefitted from your early vaccination data. This is mentioned in the repository https://github.com/lynxgav/COVID19-vaccination.

**Comment by davipt** [comment]
Hello,

The vaccination data is gathered from the official reports available at [covid19.min-saude.pt/relatorio-de-vacinacao/](https://covid19.min-saude.pt/relatorio-de-vacinacao/).

The weekly report, particularly the `dataset.csv` at the bottom of the page, was translated into the `vacinas_detalhes.csv`. It was weekly, but it had many interesting details - per age and region.
(It lacked the split per vaccine type available on the ECDC CSV file, albeit in a non-directly comparable way. ECDC measures vaccine doses, e.g., a Janssen or an infected with a single dose counts as "one dose",  but those single-dose count as "fully vaccination". It's the difference between counting vaccine doses vs. counting people.)

November 15th was the last weekly detailed report, around the day the task force got disbanded.

From the 29th, they started publishing daily PDF reports with no automatic parseable CSV. Being a PDF, we know from experience, it's going to be changing all the time. E.g., first, they reported 65-69 and later switched to 60-69 years. As it's a manual process, I've been lazily adding new columns to the `vacinas.csv` as the PDF evolves. We also have to guesstimate other values whenever possible - for example, the report only covers Portugal without islands; therefore, the value for national fully vaccinated is the official value from the PDF added to the last known value from the islands. As a result, many values are larger in reality than those on the CSV (but never overcounted).

In other words, we've reached a point on our vaccination where everyone that can be vaccinated has been vaccinated - for 12+ years old, 0.6% or about 60 thousand got no vaccination at all, and 1.4% or about 140 thousand did not get the second dose. These numbers are within a margin of error on the population count (albeit we did have census in 2021) and of people aging and crossing the age groups - or (unfortunately) dying, or getting infected and still being below the three months before the second dose. So, for example, 80+ and 70+ were already above 100% coverage due to this error between population census and reality.

I assume that's why the authorities are now focusing on the numbers that matter at this stage - 3rd dose boosters, flu shots, and the opening to 5-11 years olds. First doses and second doses are just dozens, maybe a couple of hundreds of people per day, accounting for people who finally accepted the vaccine and for those who crossed the three months after infection — a tiny percentage.

In conclusion, I've tried to keep the core values up-to-date - doses, people with at least one dose, people fully vaccinated, people with boosters - which are the values used, e.g., by Our World in Data (and then Financial Times and others) plus the VaccinationEU Twitter bot (also https://blasf1.github.io/VaccinatEU/ ).
At this point, there's not much information available from the authorities anymore.

I hope this explains the context we're in at the moment.

Thank you.
